tests/kernel/boot_ltp.pm

```diff
@@ -41,6 +40,25 @@ sub run {
     }
 
     select_serial_terminal;
+}
+
+sub run {
+    my ($self) = @_;
+    my $cmd_file = get_var('LTP_COMMAND_FILE') || '';
+
+    boot_system($self);
+
+    my $ltp_selinux = get_var('LTP_SELINUX');
+    if (defined($ltp_selinux)) {
+        if ($ltp_selinux eq 'disabled') {
+            assert_script_run("sed -i -e 's|^SELINUX=.*|SELINUX=disabled|g' /etc/selinux/config");
+            power_action('reboot', textmode => 1);
+            boot_system($self);
+        } else {
+            script_run("setenforce ${ltp_selinux}");
+        }
+        record_info("SELinux", script_output('sestatus'));
```

Comment:
I suggest move this line out of "if", then we can see status without any condition.

Reply:
No, it would fail on old systems where is not SELinux at all and I'm not interested in them.

Reply:
I will try to add it there in simple way, let's see if it is good solution.